vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -159,6 +175,9 @@
             New time slices are processed in the order of the collection, which especially matters if some of the specified change periods overlap.</String>
           </Annotation>
         </Parameter>
+        <Parameter Name="deltaIntervals" Type="Collection(Temporal.Period)" Nullable="false">
+          <Annotation Term="Core.Description" String="Periods in 1:1 correspondence with `deltaTimeslices` in case the collection to be modified does not have visible data history" />
+        </Parameter>

```

Comment:
That is too error prone: better have one input collection with both the period and the data